### PR TITLE
Bump RoC & Remove DIM dep

### PR DESCRIPTION
**Pull request by kostorr**

### CODE REVIEW

**Comment by kostorr** [comment]
bump

**Comment by ktf** [comment]
@Barthelemy can you take a look at this?

**Comment by Barthelemy** [comment]
sure, on it

File: readoutcard.sh [hunk 1]
```diff
@@ -52,7 +50,6 @@ module load BASE/1.0                                                          \\
             Common-O2/$COMMON_O2_VERSION-$COMMON_O2_REVISION                  \\
             InfoLogger/$INFOLOGGER_VERSION-$INFOLOGGER_REVISION               \\
             ${PDA_VERSION:+PDA/$PDA_VERSION-$PDA_REVISION}                    \\
```
Comment (Barthelemy) [inline readoutcard.sh]: There is a spurrious "\\" at the end of the line.

Comment (Barthelemy) [reply readoutcard.sh]: I meant `\\`

**Pull request merged**